src/hooks/layout/useScrollToTopOnFocus.ts

```diff
@@ -17,17 +17,17 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
 import { useNavigation } from '@react-navigation/native'
-import { RefObject, useEffect } from 'react'
-import { FlatList, ScrollView } from 'react-native'
+import { useEffect } from 'react'
 
+import { ScrollableViewRef } from '~/contexts/ScrollContext'
 import { scrollScreenTo } from '~/utils/layout'
 
-const useScrollToTopOnFocus = (viewRef: RefObject<ScrollView> | RefObject<FlatList>) => {
+const useScrollToTopOnFocus = (viewRef: ScrollableViewRef) => {
```

Comment:
I wanted to mention this before as well but I forgot: I find the name of this function a bit confusing when I look at its implementation. From what I read in the docs, `blur` is the event fired when focus is **removed** from a screen. Why did you go for this naming, what am I missing?

Reply:
That's absolutely correct :P I wanted to scroll to top on focus initially, but ended up changing the event to blur while forgetting to update the fn name 😆